great_expectations/core/usage_statistics/anonymizers/datasource_anonymizer.py

```diff
@@ -66,12 +48,13 @@ def _anonymize_datasource_info(self, name: str, config: dict) -> dict:
                 config=execution_engine_config,
             )
             data_connector_configs = config.get("data_connectors")
-            anonymized_info_dict["anonymized_data_connectors"] = [
-                self._aggregate_anonymizer.anonymize(
-                    name=data_connector_name, config=data_connector_config
-                )
-                for data_connector_name, data_connector_config in data_connector_configs.items()
-            ]
+            if data_connector_configs:
```

Comment:
This was a bug, we allow no data connectors in our configs but were failing here if there were no data connectors.

Reply:
Thank you for fixing this @anthonyburdi